jetty-runner/pom.xml

```diff
@@ -35,21 +35,42 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-          <groupId>org.apache.felix</groupId>
-          <artifactId>maven-bundle-plugin</artifactId>
-          <configuration>
-            <!-- jetty-runner is not an OSGi component -->
-            <skip>true</skip>
-          </configuration>
-        </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-jar-plugin</artifactId>
+        <artifactId>maven-invoker-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>integration-test</id>
+            <phase>integration-test</phase>
+            <goals>
+              <goal>install</goal>
+              <goal>integration-test</goal>
+              <goal>verify</goal>
+            </goals>
+          </execution>
+        </executions>
         <configuration>
-         <archive>
-            <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
-         </archive>
+          <javaHome>${java.home}</javaHome>
+          <environmentVariables>
+            <JAVA_HOME>${java.home}</JAVA_HOME>
+          </environmentVariables>
+          <debug>${it.debug}</debug>
+          <addTestClassPath>true</addTestClassPath>
+          <projectsDirectory>src/it</projectsDirectory>
+          <timeoutInSeconds>600</timeoutInSeconds>
+          <cloneProjectsTo>${project.build.directory}/it</cloneProjectsTo>
+          <pomIncludes>
+            <pomInclude>*/pom.xml</pomInclude>
+          </pomIncludes>
+          <localRepositoryPath>${project.build.directory}/local-repo</localRepositoryPath>
+          <settingsFile>src/it/settings.xml</settingsFile>
```

Comment:
ew!

Reply:
we use a settings file which simply use your local repo as a mirror (so we do not download all the internet again :) )
same configuration as jetty-maven-plugin